server/src/Icepeak/Server/Config.hs

```diff
@@ -121,6 +128,11 @@ configParser environment = Config
         metavar "WS-PONG-TIMEOUT" <>
         value 30 <>
         help "The timespan in seconds after sending a ping during which the client has to respond with a pong. If no pong is sent within this timeframe then the connection is considered to have timed out and it will be terminated.")
+  <*> option auto
+       (long "first-subscription-deadline-timeout" <>
+        metavar "MICROSECONDS" <>
+        value 100_000 <> -- 0.1 seconds
```

Comment:
0.1 seconds can be quite short for network connections over the internet, something like 1 or 2 seconds is probably a better default.